File: src/_1ProgrammingBasicsWithJavaMay2023/_5WhileLoop/_1Lab/_06MaxNumber.java
package _1ProgrammingBasicsWithJavaMay2023._5WhileLoop._1Lab;

import java.util.Scanner;

public class _06MaxNumber {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String input = scanner.nextLine();
        int maxNumber = Integer.MIN_VALUE;
        while (!input.equals("Stop")) {
            int number = Integer.parseInt(input);
            if (number > maxNumber) {
                maxNumber = number;
            }
        input = scanner.nextLine();
        }
        System.out.printf("%d",maxNumber);
    }
}

File: src/_1ProgrammingBasicsWithJavaScriptNovember2024/_5WhileLoop/_1Lab/_06MaxNumber.js
function maxNumber(input) {
    let command = input[0];
    let highestNumber = Number.MIN_SAFE_INTEGER;

    let counter = 1;
    while (command !== "Stop") {
        const number = Number(command);

        if (number > highestNumber) {
            highestNumber = number;
        }

        command = input[counter++];
    }

    console.log(`${highestNumber}`);
}